ffmpeg_plugin/0001-Add-ability-for-ffmpeg-to-run-svt-vp9.patch

```diff
@@ -51,13 +52,32 @@ index 34c2adb4a4..ea125e68ae 100755
 @@ -6299,6 +6302,7 @@ enabled libvpx            && {
      fi
  }
-
+ 
 +enabled libsvtvp9         && require_pkg_config libsvtvp9 SvtVp9Enc EbSvtVp9Enc.h eb_vp9_svt_init_handle
  enabled libwavpack        && require libwavpack wavpack/wavpack.h WavpackOpenFileOutput  -lwavpack
  enabled libwebp           && {
      enabled libwebp_encoder      && require_pkg_config libwebp "libwebp >= 0.2.0" webp/encode.h WebPGetEncoderVersion
+diff --git a/fftools/ffmpeg.c b/fftools/ffmpeg.c
+index 01f0410..0e2a73b 100644
+--- a/fftools/ffmpeg.c
++++ b/fftools/ffmpeg.c
+@@ -764,9 +764,11 @@ static void write_packet(OutputFile *of, AVPacket *pkt, OutputStream *ost, int u
+         if (pkt->dts != AV_NOPTS_VALUE &&
+             pkt->pts != AV_NOPTS_VALUE &&
+             pkt->dts > pkt->pts) {
+-            av_log(s, AV_LOG_WARNING, "Invalid DTS: %"PRId64" PTS: %"PRId64" in output stream %d:%d, replacing by guess\n",
+-                   pkt->dts, pkt->pts,
+-                   ost->file_index, ost->st->index);
++            if(!pkt->flags & AV_PKT_FLAG_SVT_VP9_EXT_ON) {
++                av_log(s, AV_LOG_WARNING, "Invalid DTS: %"PRId64" PTS: %"PRId64" in output stream %d:%d, replacing by guess\n",
```

Comment:
```suggestion
+                av_log(s, AV_LOG_WARNING, "Invalid DTS: %" PRId64 " PTS: %" PRId64 " in output stream %d:%d, replacing by guess\n",
```
As a quick note, you might want to put a space between the macro strings for the identifier as clang in C++ mode will reject this

Reply:
Hi @1480c1 ，
Thanks for your review.
This log is the origin code of FFmpeg, not added by the plugin patch.
This patch just add a if statement to avoid unnecessary logs.
We didn't revise the log format and style.
And there is similar code around this log in FFmpeg.c, we'd better not revise it.